Docker/README.md

```diff
@@ -32,64 +67,63 @@ Text line 2
 EOF
 ```
 
-This is used to create the file `fileToCreate` and put the following text lines until the EOF keyword.
+This command will create the text file `fileToCreate` and populate it with the lines of text that follow up but not including the EOF keyword.
 
-You can show the content of the created file using: `cat fileToCreate`
+You can display the content of the created file with the command `cat fileToCreate`.
 
-In order to append text to the file, the first `>` can be replaced with `>>`.
+In order to append text to the file, the first `>` can be replaced with `>>`.  **N.B.** If you only have one greater than character '>' the command will overwrite the file rather than appending to the file.
 
```

Comment:
Well, the Lab is not targeting Linux newbies in fact. I don't want to add too many instructions which should be pretty obvious. Also during the delivery of the Lab (because that's how I use it), I want to see who is comfortable and who is less. So I know who to help for what.